www/layouts/home.html

```diff
@@ -85,7 +85,7 @@ <h2>Your Donation Makes a Difference</h2>
           href="https://giving.mit.edu/give/to/ocw/?utm_source=ocw&utm_medium=homepage_banner&utm_campaign=nextgen_home"
           >
           <span class="h1 m-0 pr-2">GIVE</span>
-          <img src="/images/heart-orange.png" alt="heart" />
+          <img src="/images/heart-orange.png" alt="" />
```

Comment:
@pdpinch Currently, all the social (fb, insta etc) icon logo and sponsor logo images have a 1-word alt text. Should we remove that too or add aria-hidden?

Reply:
No. We need the alt text for the social icons because those are buttons and without the alt text, the screen reader user won't know they are there and what they are for. 

The social logos are not "decorative" unlike, for example, this heart.